docs/general/Mainnet/node_level_architecture.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 2
+---
+
+# Node-level architecture
+
+## Node Types
+
+### Full node
+
+A full node provides the entire functionality of a Movement node:
+
+- ingresses transactions into a local mempool
+- forms proto-blocks from the transactions in the mempool and forwards them
```

Comment:
what is a proto-block exactly?  Is this is block as part of a chain, header etc, or an ordered list of transactions or batch?

Reply:
thx this is wrong. the set of transactions sent to the DA via the DA-light-client are called blobs. The ordered set of transactions read from the DA is called protoblobs (currently have 15seconds intervals due to the frequency of celestia tendermint). i have fixed the comments.